Formula/e/emqx.rb

```diff
@@ -1,9 +1,8 @@
 class Emqx < Formula
   desc "MQTT broker for IoT"
   homepage "https://www.emqx.io/"
-  # TODO: Check if we can use unversioned `erlang` at version bump
```

Comment:
[Not yet.](https://github.com/Homebrew/homebrew-core/actions/runs/6045759248/job/16406380378#step:3:48)

```
Unsupporetd Erlang/OTP version 26.0.2
```